src/field_specials.c

```diff
@@ -4051,7 +4051,7 @@ void UpdateTrainerFanClubGameClear(void)
 
 u8 sub_813BADC(u8 a0)
 {
-    static const u8 gUnknown_085B3470[] = { 0x02, 0x01, 0x02, 0x01 };
+    static const u8 gUnknown_085B3470[] = { 2, 1, 2, 1 };
 
```

Comment:
This and a lot of the below was initial cleanup for Trainer Fan Club doc before I got sidetracked by Contest